core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -56,8 +51,12 @@ public SetupWizard(Jenkins j) throws IOException {
             HudsonPrivateSecurityRealm securityRealm = new HudsonPrivateSecurityRealm(false, false, null);
             j.setSecurityRealm(securityRealm);
             String randomUUID = UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH);
-            admin = securityRealm.createAccount(SetupWizard.initialSetupAdminUserName, randomUUID);
-            admin.addProperty(new SetupWizard.AuthenticationKey(randomUUID));
+            
+            // create an admin user
+            securityRealm.createAccount(SetupWizard.initialSetupAdminUserName, randomUUID);
+            
+            // JENKINS-33599 - write to a file in the jenkins home directory
```

Comment:
@jtnord 

>  I beleive the jenkins HOME will be 700 jenkins:jenkins anyway

Not AFAICT:
https://github.com/jenkinsci/packaging/blob/master/rpm/build/SPECS/jenkins.spec#L126